src/workspaceIndex.jsx

```diff
@@ -123,6 +124,16 @@ async function init() {
                       props => <ProtectedContent component={Workspace} {...props} />
                     }
                   />
+                  <Route
+                    exact
+                    path='/lw-workspace/'
+                    component={ErrorWorkspacePlaceholder}
+                  />
+                  <Route
+                    exact
+                    path='/no-workspace-access'
```

Comment:
why we don't import these paths from localconf? we have `workspaceUrl` and `workspaceErrorUrl`

Reply:
ok :+1: